src/Features/CSharp/Portable/UseExpressionBodyForLambda/UseExpressionBodyForLambdaCodeStyleProvider_Analysis.cs

```diff
@@ -38,7 +39,7 @@ private void AnalyzeSyntax(SyntaxNodeAnalysisContext context, CodeStyleOption2<E
             SemanticModel semanticModel, CodeStyleOption2<ExpressionBodyPreference> option,
             LambdaExpressionSyntax declaration, CancellationToken cancellationToken)
         {
-            if (CanOfferUseExpressionBody(option.Value, declaration))
+            if (CanOfferUseExpressionBody(option.Value, declaration, declaration.GetLanguageVersion()))
```

Comment:
interesting.  new extension you added? or did we always have this?

Reply:
Added an extension.